module/x/peggy/types/msgs.go

```diff
@@ -437,16 +437,14 @@ var (
 // EthereumBridgeBootstrappedClaim orchestrators confirm that the contract is setup on the Ethereum side and the init data.
 type EthereumBridgeBootstrappedClaim struct {
 	Nonce UInt64Nonce `json:"nonce" yaml:"nonce"`
-	Block uint64      `json:"block" yaml:"block"`
-	// AllowedValidatorSet addresses to participate
-	AllowedValidatorSet []EthereumAddress `json:"allowed_validator_set" yaml:"allowed_validator_set"`
-	// ValidatorPowers the validator's power values
-	ValidatorPowers []uint64 `json:"validator_powers" yaml:"validator_powers"`
 	// PeggyID is a random 32 byte value to prevent signature reuse
-	PeggyID []byte `json:"peggy_id,omitempty" yaml:"peggy_id"`
+	PeggyID string `json:"peggy_id" yaml:"peggy_id"`
```

Comment:
So serializing and signing the bytes encoding for Amino is a little difficult on the Rust side. Especially since Amino's days are numbered I have decided not to fix it and instead prefer to use strings. We can evaluate switching back after we move to Protobuff.